src/MinterGateway.sol

```diff
@@ -881,12 +877,13 @@ contract MinterGateway is IMinterGateway, ContinuousIndexing, ERC712Extended {
      * @param newTimestamp_ The timestamp of the collateral update.
      */
     function _updateCollateral(address minter_, uint240 amount_, uint40 newTimestamp_) internal {
-        MinterState storage minterState_ = _minterStates[minter_];
-
-        uint40 lastUpdateTimestamp_ = minterState_.updateTimestamp;
+        uint40 minimumTimestamp_ = UIntMath.max40(
+            _minterStates[minter_].updateTimestamp,
+            uint40(block.timestamp) - updateCollateralInterval()
+        );
 
-        // Revert if minter's last collateral update is more recent.
-        if (newTimestamp_ <= lastUpdateTimestamp_) revert StaleCollateralUpdate(newTimestamp_, lastUpdateTimestamp_);
+        // A more recent collateral update already exists or the the update is not recent enough.
+        if (newTimestamp_ <= minimumTimestamp_) revert StaleCollateralUpdate(newTimestamp_, minimumTimestamp_);
```

Comment:
Let's say the `updateCollateralInterval()` is 24 hours, I've updated collateral at time T0 = 0, so `_minterStates[minter_].updateTimestamp = 0`.
I update again at T2 = 26 hours, so current `block.timestamp = 25` and `uint40(block.timestamp) - updateCollateralInterval() = 2`.
So `minimumTimestamp_ = 2`.
The validators `minTimestamp` was taken at T1 = 1 hour, which means that  `newTimestamp_ = 1`.
I'm not able to update collateral with these signatures and need to provide more recent ones.
Which means I can't update my collateral outside of the interval by providing valid signatures that were signed in the `updateCollateralInterval`.

So in this case, what prevails is the time at which the update collateral is performed but not the time at which the signatures were signed.

Reply:
I don't follow. This prevent the `minTimestamp` from all validators from being older than an update interval ago.